### DISCUSSION TITLE
动态又有了新的广告，这个之后可以过滤掉吗

### DISCUSSION THREAD

**Discussion by Alexius66** [the1812/Bilibili-Evolved#3778] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/73510996/197502133-0ee8c26f-1461-4bbb-ac84-6311bca8e419.png)

**Comment by Geeyun-JY3** [comment]
这玩意挺久了的吧，就是给 UP 恰饭的：https://github.com/the1812/Bilibili-Evolved/issues/731#issuecomment-707136216

**Comment by Tinhone** [comment]
https://t.bilibili.com/710540809598926854
class="bili-dyn-content__orig__additional"
